server/datastore/mysql/migrations/tables/20240826111228_AdjustHostIndexes.go

```diff
@@ -0,0 +1,41 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20240826111228, Down_20240826111228)
+}
+
+func Up_20240826111228(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+	    ALTER TABLE hosts
+	    DROP INDEX host_ip_mac_search
```

Comment:
IIRC there were some weirdness around searching IP addresses back in the day. Did you look into when/why this index was added in the first place?

Reply:
This index was not being used _at all_ because it's `FULLTEXT`, which requires special syntax to search, 100% safe to remove.

Reply:
Makes sense to remove it. Should we open a follow up ticket to confirm that IP search is working as expected, i.e. that whatever motivated the addition of the index isn't an issue (assuming it was implemented properly in the first place)?

Reply:
I think it was implemented without knowing about the syntax. It really wasn't used so I don't see the need to go further down!

Reply:
Ok, looks like it was added in March 2020, which is before the company was incorporated!

It seems like it was implemented with the MATCH/AGAINST [syntax](https://github.com/fleetdm/fleet/commit/ea93843203e04c193e973d9f9067cafc2959261a#diff-ec797a071df046dfb849880d689b5dc274601d19626e17e2f61e6ce663adaea9R327) but must have been broken along the way.